mkosi/resources/mkosi-initrd/mkosi.conf.d/10-debian-ubuntu.conf

```diff
@@ -17,10 +17,10 @@ Packages=
 
         # Various libraries that are dlopen'ed by systemd
         libfido2-1
-        ^libtss2-esys-[0-9\.]+-0$
-        libtss2-rc0
-        libtss2-mu0
-        libtss2-tcti-device0
+        ?name(^libtss2-esys-[0-9\.]+-0$)
+        ?exact-name(libtss2-rc0)
+        ?name(libtss2-mu[0-9\.\-]+)
```

Comment:
`.` and `-` do not need to be escaped here. Remove the backslashes.

Reply:
True, I removed it. Let's see whether it works, the regex implementation seems a bit weird, e.g it doesn't support groups, so choices between alternatives can't really be made.